lib/aws/ses.rb

```diff
@@ -0,0 +1,27 @@
+##
+# ActionMailer delivery method for SES inspired by https://github.com/drewblas/aws-ses
+#
+module Aws
+  module SES
+    class Base
+      def initialize(*args); end
+
+      def deliver!(mail)
+        response = ses_client.send_raw_email(
+          raw_message: {
+            data: mail.to_s,
+          },
+          destinations: [mail.to, mail.cc, mail.bcc].flatten.compact
+        )
+        mail.message_id = "#{response.message_id}@email..amazonses.com"
+        response
+      end
+
+      private
+
+      def ses_client
+        @ses_client ||= Aws::SES::Client.new(region: Figaro.env.aws_region)
```

Comment:
We shouldn't need to provide any creds here since we can use [AWS instance profiles](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html).

With that in mind, we'll want to hold off on merging this until we have an issue resolved with devops to grant the necessary SES permissions to the IAM role associated with the EC2 instance profiles.